change-the-middle-management-mindset.md

```diff
@@ -0,0 +1,83 @@
+* Fellows program if you can achieve guest TC; you get one day a week to do cross-platform work (they report on it as a measurement of success rather than a measurement of loss). Planning is good as a part of resource allocation (might have to change expectations; we're one team, a whole group).
+* Champions program recruited from any group: could be a Middle-Management champion who has gone through the process before. See who are the new believers. Could put badges on people's names in the directory. Could do cardboard cut-outs of the champion with an IS t-shirt ;-)
+
+* Built-into the whole design of IS process. Delivery teams don't own any code themselves; they change code amongst the product teams. Middle Management for product teams know that requests will come in for their code to be changed. They need to have their developers provide mentorship to the delivery team developers. We have different BLs; each is represented by a delivery team. Product teams focus on larger architectural decisions, ensure that the delivery teams don't mess too much with the Products.
+    - We had globalization teams responsible for countries; had constraints (compliance) for each country. Those teams are always good about IS. But for some reason they went away (restructuring).
+
+* Architectural solutions
+    - Microservices architecture: creates incentive organizationally for people to collaborate. If a bug occurs, then it creates a problem for the users. 
+        - but it might be violating an SLA if the bug isn't being fixed
+        - the problem comes with feature requests or affects design (going beyond bug fixes)
+    - Developing platforms is an ideal IS use case (hackathon to build applications on top)
+    - SW Architects have to have an IS mentality and work together.
+        - newer companies with open source developers have that mindset
+
+* Empowering Middle Management - IS readiness checklist; Middle Management should partner with their developers. What are the opportunities out there. Can we come up with justification for you to spend any time on this (how does this tie together with our KPIs)
+
+## Resulting Context
+* **Support for IS is automatic, standard and expected from Middle Management.**
+* More measurement is required; measurement becomes more sophisticated (easier to measure your own stuff; harder to measure others). Keeping track of time spent in projects.
+* Better training for Middle Management in negotiation and facilitation will be needed.
+* Engineering cost vs. benefit - you will support others; other teams will allow you to make PRs and this will save you time (a balance in the long run.)
+
+## Known Instances
+
+
+## Sketch
+![How to help Middle Managers actively support InnerSource projects involving their people](http://teiru.net/images/middle_management_sketch_img_2257.jpg)
+
+## Authors
+* Silona Bonewald
```

Comment:
This is the first time _silo_ is mentioned in this pattern ... interesting. No offence, @Silona ;)